openssl/src/ocsp.rs

```diff
@@ -17,7 +17,7 @@ use openssl_macros::corresponds;
 bitflags! {
     #[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
     #[repr(transparent)]
-    pub struct OcspFlag: c_ulong {
+    pub struct OcspFlag: c_int {
```

Comment:
I believe this is also technically backwards incompatible. What's the motivation?